src/run_environment/reference_genome.ml

```diff
@@ -175,6 +175,9 @@ module Default = struct
     let dbsnp_mm10_indels_url =
       "ftp://ftp-mouse.sanger.ac.uk/REL-1303-SNPs_Indels-GRCm38/\
        mgp.v3.indels.rsIDdbSNPv137.vcf.gz" in
+    let gene_annotations_gtf =
+      "ftp://ftp.ensembl.org/pub/release-79/gtf/mus_musculus/\
```

Comment:
Pretty arbitrary; we're using 79 in other places. We need a better story when it comes to minor releases of the same genome...